### PR TITLE
Newsletter draft for December 2021

### PR DESCRIPTION
**Pull request by milancurcic**
Here's the draft. Please add any events that I missed.

* [x] Update PR data on Nov 30
* [x] Update contributors data on Nov 30
* [x] Flang update from @AlexisPerry
* [x] LFortran update from @certik

Thank you!

### CODE REVIEW

File: _posts/2021-12-01-Fortran-Newsletter-December-2021.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+  Add: workflow to make installer on push and release
+* [#614](https://github.com/fortran-lang/fpm/pull/614):
+  Bump version to 0.5.0
+* [#7](https://github.com/fortran-lang/setup-fpm/pull/7) (`setup-fpm`):
+  Fix Latest Option
+
+### Work in progress
+
+* [#622](https://github.com/fortran-lang/fpm/pull/622) (WIP):
+  (WIP) Cleanup the backend output
```
Comment (awvwgk) [inline _posts/2021-12-01-Fortran-Newsletter-December-2021.md:133]:
```suggestion
  Cleanup the backend output
```

**Comment by AlexisPerry** [comment]
PR created for Flang updates: https://github.com/milancurcic/fortran-lang.github.io/pull/2

**Comment by certik** [comment]
I don't have the time to write it up for LFortran. If anybody has time to help, I would appreciate it. If not, then let's skip this month.

**Comment by milancurcic** [comment]
No problem, I should be able to write it up this afternoon.

**Comment by milancurcic** [comment]
@certik see 64e5950310deb3076f36cbff21e7d6e2b7ce9988. I only summarized user-facing changes for brevity. I think this newsletter is ready to go.

File: _posts/2021-12-01-Fortran-Newsletter-December-2021.md [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+
+The candidate for file system operations to be included in stdlib is being developed by
+[@MarDiehl](https://github.com/MarDiehl) and [@arjenmarkus](https://github.com/arjenmarkus)
+in [this repository](https://github.com/MarDiehl/stdlib_os).
+Please try it out and let us know how it works, if there are any issues, or if the API can be improved.
+
+## Fortran Package Manager
+
+Here's what's new in fpm:
+
```
Comment (awvwgk) [inline _posts/2021-12-01-Fortran-Newsletter-December-2021.md:110]: Let's highlight the fpm release as well:
```suggestion

* [v0.5.0](https://github.com/fortran-lang/fpm/releases/tag/v0.5.0):
  Alpha release update
```

File: _posts/2021-12-01-Fortran-Newsletter-December-2021.md [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+  Repair --list option and correct obsolete descriptions of the --list option
+* [#612](https://github.com/fortran-lang/fpm/pull/612):
+  Fix modules listing (for install)
+* [#613](https://github.com/fortran-lang/fpm/pull/613):
+  Add: critical section to mkdir in backend
+* [#616](https://github.com/fortran-lang/fpm/pull/616):
+  Add: workflow to make installer on push and release
+* [#614](https://github.com/fortran-lang/fpm/pull/614):
+  Bump version to 0.5.0
+* [#7](https://github.com/fortran-lang/setup-fpm/pull/7) (`setup-fpm`):
```
Comment (awvwgk) [inline _posts/2021-12-01-Fortran-Newsletter-December-2021.md:127]: Maybe prefixing with the repo name is easier to distinguish
```suggestion
* [setup-fpm#7](https://github.com/fortran-lang/setup-fpm/pull/7):
```

**Review by awvwgk — APPROVED** [review]

**Comment by milancurcic** [comment]
I'll go ahead and merge. Thank you all!

**Comment by certik** [comment]
@milancurcic thanks Milan! I really appreciate it. It looks great.

**Pull request merged**